core/modules/replica/QservMgtServices.h

```diff
@@ -207,6 +208,28 @@ class QservMgtServices
                                             std::string const& jobId="",
                                             SetReplicasQservMgtRequest::CallbackType onFinish = nullptr,
                                             unsigned int requestExpirationIvalSec=0);
+
+    /**
+     * Set new replicas at the new replicas at a Qserv worker
```

Comment:
This comment looks like it was just copied from previous method.

Reply:
👍